README.md

```diff
@@ -31,7 +31,9 @@ $ git clone https://github.com/infracloudio/botkube.git
 If you are not interested in events about particular resource, just remove it's entry from the config file.
 - Deploy BotKube controller using **helm install** in your cluster.
 ```bash
-$ helm install --name botkube --namespace botkube --set config.communications.slack.channel={SLACK_CHANNEL_NAME},config.communications.slack.token={SLACK_API_TOKEN_FOR_THE_BOT},config.settings.clustername={CLUSTER_NAME},config.settings.allowkubectl={ALLOW_KUBECTL} helm/botkube/
+$ helm install --name botkube --namespace botkube --set 
```

Comment:
Lets make this more readable as in botkube docs.
 Ideally we should remove installation steps from README and just rely on botkube docs

Reply:
Updated the helm command in README.